content/post/Android_Root101.en.md

```diff
@@ -1,31 +1,32 @@
 +++
 title = "A Guide to Flashing Custom ROMs [Android]"
-slug = "Flashing Custom ROMs 101"
+slug = "csrom101"
 date = "2018-02-25T00:00:00"
+image = "img/rohanvtk/csrom.jpg"
 description = "Sick and tired of the bloatware on your current OEM ROM. Head over and learn how to flash custom ROMs."
 author = "RohanVTK"
-tags = ['Smartphone', 'Customization']
+tags = ['Smartphone', 'Customization','Android']
 categories = ['Android']
 +++
 
-**_(This blog starts off with a warning: Although the process is super easy and there hardly is a chance that you will brick your device during the process but, it's my responsibility to make you aware of it)_**
+**_(This blog starts off with a warning: Although the process is super easy and the chances of your phone getting bricked(useless) are negligible but, it's my responsibility to make you aware of it)_**
```

Comment:
```suggestion
**_This blog starts off with a warning: Although the process is super easy and the chances of your phone getting bricked(useless) are negligible but, it's my responsibility to make you aware of it_**
```
Don't need parenthesis,
a warning is not secondary stuff, it's an important part